README.md

```diff
@@ -9,11 +9,12 @@ produced from the Rust code using [Rust WASM](https://rustwasm.github.io/docs/bo
 The TypeScript library is experimental. Over time, it might replace the [bitbox02-api NPM
 package](https://www.npmjs.com/package/bitbox02-api).
 
-# Rust
+## Rust (required)
 
-Install the protoc protobuf compiler, e.g. on Ubuntu:
+Install the protoc protobuf compiler:
 
-    sudo apt-get install protobuf-compiler
+    Ubuntu: sudo apt-get install protobuf-compiler
+    MacOS: brew install protobuf
```

Comment:
normally code/shell commands are on single lines indented by spaces so they get highlighted and are easy to copy. pls split into multiple lines, like

On Ubuntu:

    ....


On macOS:

    ....

Reply:
fixed